py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -133,7 +139,7 @@ def check_response(self, response):
                     pass
 
         if status in ErrorCode.NO_SUCH_ELEMENT:
-            exception_class = NoSuchElementException
+            exception_class: Type[WebDriverException] = NoSuchElementException
```

Comment:
Does line 144 need this hint too?

Reply:
And carried on through the file

Reply:
@AutomatedTester the type hint will be applied to all subsequent assignments. I agree though this type declaration is misleading and unclear; I've moved it to be in a separate line before the status checks. The line will be ignored by the interpreter and is only used for type hinting.